docs/symptom-survey/coding.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Questions and Coding
+parent: COVID Symptom Survey
+nav_order: 3
+---
+
+# Questions and Coding
+{: .no_toc}
+
+The symptom surveys have been deployed in several waves. We have tried to ensure
+the coding of waves is consistent. This page provides the full survey text and
+coding schemes.
+
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Basic Coding Rules
+
+All choice responses are recorded numerically starting from 1, in displayed
+order from left to right and top to bottom. When the respondent is allowed to
+select multiple responses, these are shown separated by commas, such as `2,4,6`.
+Questions left blank or with invalid responses are recorded in the CSV files as
+`NA`.
+
+Timestamps are provided in Pacific time (UTC-7). The following metadata columns
+describe each survey response:
+
+* `StartDatetime`: The time the respondent began the survey.
+* `EndDatetime`: The time the respondent's responses were recorded. This either
```

Comment:
No, that would be "RecordedDate" which is not included in these files.

"EndDatetime" is the last time of any respondent activity, which may be the same as the recorded time or four hours before the recorded time.